pkg/pxc/app/statefulset/haproxy.go

```diff
@@ -66,8 +66,8 @@ func (c *HAProxy) AppContainer(spec *api.PodSpec, secrets string, cr *api.Percon
 				Name:          "mysql-replicas",
 			},
 			{
```

Comment:
I believe we should left 3309 for proxy traffic

Reply:
done